hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1568,14 +1568,14 @@
             - Write the KEY_IV_DATA_IN_CLEAR and DATA_OUT_CLEAR trigger bits to 1 and wait for it to
               complete by polling the status idle bit.
             - Read back the data out CSRs - they should all read garbage values.
-            - Assertion check verifies that the internal states (data_in, key share and IV are also
-              garbage, i.e. different from the originally written values.
+            - Assertion check verifies that the IV are also garbage, i.e. different from the originally
+              written values.
             - Assertion checks proves that all interfaces are connected across AST RNG, ES, CSRNG,
               EDN and AES.
             - Predict the generated entropy bits and check against the observed for correctness.
```

Comment:
@vogelpi could you please advise me how to do this last step?

Reply:
This last step is very difficult if not impossible and some time ago we did re-discuss this (see forwarded email thread). The conclusion was:
1. Between AST (generating the raw entropy) and consumers like AES there sit several modules such as entropy_source, CSRNG and EDN. Predicting what AST generates and how its output travels through system is very challenging to predict.
1. Even though it would be great to have this capability and we originally discussed discussed the idea (that's why it ended up in the testplan), we concluded not to do it in the end. It seems we forgot to update the testplan accordingly.
1. Instead, what we want is assertion checks (these are most likely SystemVerilog assertions) to ensure things are wired up correctly. I.e., we want to check that what leaves EDN ends up at the AES input eventually, but we don't need to track how the randomness evolves inside EDN or AES. This is part of the IP-level tests.

Please correct me if I am wrong @sriyerg , @rasmus-madsen , @mwbranstad , @martin-lueker

Reply:
This is correct.  Even at the entropy_src level, there is some trouble predicting the exact output when the AST stand-in is known (see issue https://github.com/lowRISC/opentitan/issues/10092).   The scheme advised by @vogelpi is much more feasible.